src/open_inwoner/templates/pages/kvk/branches.html

```diff
@@ -2,36 +2,63 @@
 {% load i18n static form_tags grid_tags button_tags card_tags button_tags %}
 
 {% block content %}
-<p> {% trans "Selecteer de vestiging waarmee u wilt inloggen" %}:</p>
-{% render_grid %}
-    {% render_column start=0 span=6 %}
-        <form method="POST" id="eherkenning-branch-form">
-            {% csrf_token %}
-            {% if form.branch_number.errors %}
-                {% errors errors=form.branch_number.errors %}
-            {% endif %}
-            {% for branch in company_branches %}
-                {% render_card direction='horizontal' %}
-                    {% with company_id=branch.vestigingsnummer %}
-                        <label class="left" for="branch-{{ company_id }}">
-                            <p>{{ branch.naam }}</p>
-                            {% if branch.straatnaam and branch.plaats %}
-                                <p>{{ branch.straatnaam }}, {{ branch.plaats}}</p>
-                            {% endif %}
-                            {% if branch.vestigingsnummer %}
-                                <p>{% trans "Vestiging" %}: {{ company_id }}</p>
-                            {% endif %}
-                        </label>
-                        <input type="radio" id="branch-{{ company_id }}" name="branch_number" value="{{ company_id }}">
+    {% render_grid %}
+        {% render_column start=4 span=6 %}
+            <h1 class="h1" id="title">{% trans "Log in" %}</h1>
```

Comment:
Are we not using `utrecht-heading-1` for all `h1` elements?

Reply:
Yes, although I'd like to change NL Design-System stuff in a completely new task/sprint, but I can change this here too.